ocs/constants.py

```diff
@@ -43,3 +43,7 @@
 # encoded value of 'admin'
 ADMIN_BASE64 = 'YWRtaW4='
 GB = 1024 ** 3
+
+# component
+RBD_COMPONENT = 'rbd'
```

Comment:
INTERFACE?
I asked ceph ppl how they used to call the ceph backend type and been told 'interface'